metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -1168,6 +1213,12 @@ def make_lineage_dataset_urn(self, platform: str, name: str) -> str:
             platform, name, platform_instance, self.config.env
         )
 
+    def get_platform_instance_id(self) -> str:
+        return self.config.platform_instance or self.platform
+
+    def close(self):
```

Comment:
I think you don't need this if you inherit from `StatefulIngestionSourceBase` as it has this close

Reply:
yup

Reply:
Done